src/server/controller/cycleData/updateOriginalDataPoint/index.ts

```diff
@@ -16,11 +17,13 @@ export const updateOriginalDataPoint = async (
 ): Promise<OriginalDataPoint> => {
   const { assessment, cycle, originalDataPoint, user } = props
 
+  const updatedOriginalDataPoint = await OriginalDataPointRepository.update(
+    { assessment, cycle, originalDataPoint },
+    client
+  )
+
   return client.tx(async (t) => {
-    const updatedOriginalDataPoint = await OriginalDataPointRepository.update(
-      { assessment, cycle, originalDataPoint },
-      t
-    )
+    await updateDependantNodes({ originalDataPoint: updatedOriginalDataPoint, assessment, cycle, user }, t)
```

Comment:
cant we just reuse the logic from src/server/controller/cycleData/persistNodeValue/index.ts line 18-31? why reimplementing the logic here?

Reply:
we should talk in person